src/operate.test.js

```diff
@@ -0,0 +1,36 @@
+import Big from 'big.js';
```

Comment:
- [ ] kindly as requested by the previous reviewer [here](https://github.com/Cosmosha/Math-Magicians/pull/8#discussion_r1159732449) consider adding tests for all react components in your app because that will make your app full fill the below requirements 
> Create unit tests for all React components:
> - Use Jest snapshots to test the components.
> - Use React Testing Library to simulate user interaction.


and finally make sure All tests are passing before you go for another review